components/automate-ui/src/app/pages/integrations/detail/integrations-detail.component.html

```diff
@@ -34,16 +34,28 @@
     </chef-page-header>
 
     <div class="page-body">
+      <chef-button primary caution 
+        [disabled]="!hasSelectedNodes()" 
+        (click)="deleteNodes()">
+        Remove Nodes
+      </chef-button>
```

Comment:
I'd wrap this button in a `chef-toolbar` to ensure it has correct spacing around it, particularly its left margin:

<img width="329" alt="Screen Shot 2019-05-02 at 10 22 08 PM" src="https://user-images.githubusercontent.com/479121/57119664-357d4680-6d29-11e9-86d8-c01b8e21a9a3.png">
<img width="322" alt="Screen Shot 2019-05-02 at 10 21 27 PM" src="https://user-images.githubusercontent.com/479121/57119662-34e4b000-6d29-11e9-929d-3adbaab88874.png">

That's how 'table action' buttons are authored in some other table/list views:
[integrations-list.component.html](https://github.com/chef/automate/blob/master/components/automate-ui/src/app/pages/integrations/list/integrations-list.component.html#L17-L21)
[project-list.component.html](https://github.com/chef/automate/blob/master/components/automate-ui/src/app/pages/project/list/project-list.component.html#L18-L22)